pom.xml

```diff
@@ -1,5 +1,17 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
+    <dependencies>
+        <dependency>
+            <groupId>com.github.jnr</groupId>
+            <artifactId>jnr-constants</artifactId>
+            <version>0.9.9</version>
+        </dependency>
+        <dependency>
+            <groupId>com.github.jnr</groupId>
+            <artifactId>jnr-posix</artifactId>
+            <version>3.0.41</version>
```

Comment:
So there is an issue with that...

1) The ProcessLiveness code is being executed on Agent
2) Agent loads only one version of library to its JAR cache
3) Depending on the call, it may download an old 3.0.1 version from the core (before your patch) or the new one in the plugin

OTOH the plugin library is a priority for core side IIRC. But I would not really rely on availability of classes from 3.0.41. So I would use reflection to retrieve the stuff in `ProcessLiveness` implementation and comment out this dependency